graph/client.py

```diff
@@ -79,16 +105,37 @@ def prepare_graphs(self, rules):
             raise ValueError('Rule: "{}" Error: "{}"'.format(rule, error))
 
     def open_web_browser(self):
-        try:
-            webbrowser.get('firefox').open_new_tab(
-                'html_interpreter/index.html')
-        except BaseException:
-            webbrowser.open_new_tab('html_interpreter/index.html')
+        if not self.off_webbrowser:
+            try:
+                webbrowser.get('firefox').open_new_tab(
+                    'html_interpreter/index.html')
+            except BaseException:
+                webbrowser.open_new_tab('html_interpreter/index.html')
 
     def parse_arguments(self, args):
         parser = argparse.ArgumentParser(
             description='Client for visualization of SCAP rule evaluation results')
-
+        parser.add_argument(
+            '--show-fail-rules',
+            action="store_true",
+            default=False,
+            help='Show only FAIL rules')
+        parser.add_argument(
+            '--show-not-selected-rules',
+            action="store_true",
+            default=False,
+            help="Show notselected rules. These rules will not be visualized.")
+        parser.add_argument(
+            '--off-web-browser',
+            action="store_true",
+            default=False,
+            help="It does not start the web browser.")
+        parser.add_argument(
+            '--remove-pass-tests',
+            action="store_true",
+            default=False,
+            help=('If graph have many nodes in graph, you can remove passing'
```

Comment:
This is a weird language. "Graph has a node in graph" doesn't make sense.

What about

"Remove passing tests for better orientation in large graphs."

"Remove passing tests for better orientation in graphs that contain a large amount of nodes."

Or maybe, instead of "remove"  you can use "Do not display" as they aren't removed from anywhere, they only aren't shown.

Reply:
I will use "Do not display passing tests for better orientation in graphs that contain a large amount of nodes".